release/release_gold.sh

```diff
@@ -105,27 +105,39 @@ function tag_release {
 		repository=liferay-portal
 	fi
 
-	if (! curl
-			"https://api.github.com/repos/liferay/${repository}/git/tags" \
-			--data-raw '
-				{
-					"message": "",
-					"object": "$(lc_get_property release-data/release.properties git.hash.liferay-portal-ee)",
-					"tag": "${LIFERAY_RELEASE_VERSION}",
-					"type": "commit"
-				}' \
-			--fail \
+	if (! curl \
```

Comment:
Instead of putting if into if, exit with SKIPPED if the tag exist. If it not, then you can proceed as before. Something like this:

if (check if tag exists)
then
  return SKIPPED
fi

.... tag the release as before...

This we you get a simpler code.